client/src/components/SearchBar.js

```diff
@@ -77,7 +77,7 @@ const SearchBar = ({
   const popoverContent = (
     <AdvancedSearch
       onSearch={runAdvancedSearch}
-      onCancel={setShowAdvancedSearch}
+      onCancel={() => setShowAdvancedSearch(false)}
```

Comment:
Maybe not particularly about this line but, i can't find the code that makes `showAdvancedSearch` `true`. Do we not ever change this state's value? If not, we should remove state logic and just make it `const` and  `false`.

Reply:
Good catch
@ysf-simsoft - what do you think?
I think the intended logic seems to have been: when advanced search is shown, hide the simple search icon. We should either restore that, or remove the whole thing. I suggest to simply remove it as it is simpler from both code and UI point of view.

Reply:
It used to work like this: when advanced search is shown, hide the simple search icon.
Reason: if you open the advanced search and use some filters but don't click on the search button on the advanced search overlay but you click the search icon of the search field, it will not use the just added filters.

Reply:
And that is currently broken on `candidate`:
- open http://localhost:8080/
- verify that magnifying glass icon is shown at the end of the search text input
- click on *Add filters*: magnifying glass icon is still shown, but should be gone
- click *Cancel*: magnifying glass icon is gone, but should be shown